Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBaseTest.cs

```diff
@@ -204,7 +205,7 @@ public void TestDiagnosticMetadataRenderingInTitle ()
       var document = Html.GetResultDocument();
       var th = Html.GetAssertedChildElement (document, "th", 0);
       Html.AssertAttribute (th, DiagnosticMetadataAttributes.ItemID, Column.ItemID);
-      Html.AssertAttribute (th, DiagnosticMetadataAttributes.Content, Column.ColumnTitleDisplayValue);
+      Html.AssertAttribute (th, DiagnosticMetadataAttributes.Content, Column.ColumnTitleDisplayValue.ToString());
```

Comment:
```suggestion
      Html.AssertAttribute (th, DiagnosticMetadataAttributes.Content, Column.ColumnTitleDisplayValue.GetValue());
```

Since `DiagnosticMetadataAttributes.Content` contains the raw value of the `WebString`, you should also assert the raw value from  `Column.ColumnTitleDisplayValue`